index.html

```diff
@@ -3,12 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <title>Tamagotchi</title>
-    <link href="./styles.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=VT323&display=swap" rel="stylesheet">
+    <link href="./styles.css" rel="stylesheet"/>
   </head>
   <body>
     <div class="gameBoy">
       <div class="top"></div>
-      <div class="display"></div>
+      <div class="mid">
+        <div class="health">
+          <div class="stats"><img src="./images/Icon=Health.png" alt="health icon"><span class="number">10</span></div>
+          <div class="stats"><img src="./images/Icon=Hunger.png" alt="hunger icon"><span class="number">10</span></div>
+          <div class="stats"><img src="./images/Icon=Sleep.png" alt="sleep icon"><span class="number">10</span></div>
+          <div class="stats"><img src="./images/Icon=Fun.png" alt="fun icon"><span class="number">10</span></div>
+        </div>
```

Comment:
**praise:** HTML markup looks good, and clean. Class names are also readable, and simple